derp/derp_server.go

```diff
@@ -215,6 +225,18 @@ func (s *Server) PrivateKey() key.Private { return s.privateKey }
 // PublicKey returns the server's public key.
 func (s *Server) PublicKey() key.Public { return s.publicKey }
 
+// Returns a (possibly-reused) byte slice of the given length
+func (s *Server) empty_bytes(length uint32) []byte {
```

Comment:
`int` in the normal size used in APIs for talking about sizes of in-memory things